package.json

```diff
@@ -3559,23 +3559,23 @@
                 "command": "aws.codeWhisperer",
                 "key": "alt+c",
                 "mac": "alt+c",
-                "when": "editorTextFocus && CODEWHISPERER_ENABLED"
+                "when": "editorTextFocus && (CODEWHISPERER_ENABLED || isCloud9)"
             },
             {
                 "command": "aws.codeWhisperer.rejectCodeSuggestion",
                 "key": "escape",
                 "mac": "escape",
-                "when": "inlineSuggestionVisible && !editorReadonly && CODEWHISPERER_ENABLED"
+                "when": "inlineSuggestionVisible && !editorReadonly && (CODEWHISPERER_ENABLED || isCloud9)"
```

Comment:
parens also aren't supported in the vscode version used by cloud9.

so this is misleading (and i'm not sure if it's ignored, so it's safer to remove the parens)

Reply:
gotcha, thanks for the info Justin, let me udpate it.

Reply:
Yikes, I'm not sure if the boolean expression will be resolved correctly without brackets. Make sure to verify this.

It may equate to `(inlineSuggestionVisible && !editorReadonly && CODEWHISPERER_ENABLED) || isCloud9` if we do not have any brackets.
See https://www.emathhelp.net/en/calculators/discrete-mathematics/boolean-algebra-calculator/?f=A%E2%88%A7B%E2%88%A8C

Reply:
> It may equate to `(inlineSuggestionVisible && !editorReadonly && CODEWHISPERER_ENABLED) || isCloud9`

Then it's probably safer to put `isCloud9 || ...` at the start